### DISCUSSION TITLE
Not treat zh-tw OS as zh-cn in default setting

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#7315] [General]
Currently, RustDesk will treat an OS that using Traditional Chinese as Simplified Chinese, but the translation for Traditional Chinese is complete, so no need to use Simplified Chinese as default settings.

**Comment by rustdesk** [comment]
Please correct this, if there is something wrong.

https://github.com/rustdesk/rustdesk/blob/162039473681a1b80dc09fcf57690063849d41a4/src/lang.rs#L95

**Reply by xlionjuan** [reply]
Sorry I couldn't figure out the code, but I can sure all platforms (Windows, macOS, Android, Linux, iOS) will see same behavior.

**Comment by xlionjuan** [comment]
#7326
